pkg/datasource/formatters/json/json.go

```diff
@@ -40,6 +40,12 @@ var (
 	closerArrayPretty = []byte("\n]")
 )
 
+const (
+	// SkipFieldAnnotation is used to indicate that this field should be
+	// skipped.
+	SkipFieldAnnotation = "formatters.json.skip"
```

Comment:
Name clash 🙈 

IMHO `formatters` should be used for everything regarding the formatters operator. For this, I think `json.skip` would be good enough. WDYT?

Reply:
Makes sense.